controllers/result.go

```diff
@@ -77,6 +80,52 @@ func (c *ResultController) Create() {
 	c.Redirect("/", http.StatusFound)
 }
 
+func (c *ResultController) Show() {
+	c.EnsureAuthenticatedUser()
+	c.TplName = "results/show.html"
+	c.Data["Title"] = "Result Detail"
+	web.ReadFromRequest(&c.Controller)
+
+	resultID, err := c.getResultID()
+	if err == nil {
```

Comment:
What if the result id is not found? Should we redirect to the root path with a flash message in this case?

```Go
flash := web.NewFlash()
flash.Error(err.Error())
flash.Store(&c.Controller)
c.Ctx.Redirect(http.StatusFound, "/")
return

```

Reply:
@junan The current solution is displaying the error message on the same page with a link to the root path, do you think this works?
 
![Screen Shot 2564-06-07 at 14 30 30](https://user-images.githubusercontent.com/1772999/120976950-030e6500-c79d-11eb-9c6a-c8097186e824.png)

Reply:
Its fine to me